config/rewrites.json

```diff
@@ -12,6 +12,13 @@
             "to": "/$1/",
             "rewrite": false,
             "status": 301
+        },
+        {
+            "description": "Blog: Remove .html extension",
+            "from": "^\\/blog\\/(.+?)(\\.html)$",
```

Comment:
Sure. Both preparers currently normalize the content IDs they submit so I _think_ it'd be safe. Makes sense, though.